### PR TITLE
Feature/add consumer to schema

### PR DESCRIPTION
**Pull request by kliptonize**
See #48 
——Backend———
- Added consumer field to schema
- Added publication.from and .to upon save
- Fixed missing published status
- Added (all, currently) roles to schema. FIX THIS
——Frontend———
- Fixed small bug in icon on dashboard
- Dynamically load roles
- Add metadata (consumer) to schema
- Differ identifier field on front-end based on consumer in schema

### CODE REVIEW

**Comment by EveraertJan** [comment]
define `Added (all, currently) roles to schema. FIX THIS`?

File: images/api/src/Schema.js [hunk 1]
```diff
@@ -175,37 +174,48 @@ class Schema {
...
+          result['headers'] = temp;
 
 
-          result["created_at"] = r[0].created_at;
-          result["created_at"] = r[0].created_at;
-          result["updated_at"] = r[0].updated_at;
+          result['created_at'] = r[0].created_at;
+          result['updated_at'] = r[0].updated_at;
 
+          result['roles'] = [];
```
Comment (kliptonize) [inline images/api/src/Schema.js]: Right here, all roles are stored on a scheme, should be limlited to whatever the front-end sends, but that's not the case (yet)

**Comment by kliptonize** [comment]
Right now all roles are stored on a schema when it's created, or when the meta-data is saved. See comment on files

**Comment by EveraertJan** [comment]
Kzal er zaterdag als eerste na kijke. snot very clear

**Pull request merged**